ydb/mvp/oidc_proxy/openid_connect.cpp

```diff
@@ -227,5 +237,16 @@ TCheckStateResult CheckState(const TString& state, const TString& key) {
     return TCheckStateResult();
 }
 
+TString DecodeToken(const TStringBuf& cookie, const NActors::TActorContext& ctx) {
+    TString token;
+    try {
+        Base64StrictDecode(cookie, token);
+    } catch (std::exception& e) {
+        LOG_DEBUG_S(ctx, EService::MVP, "Base64Decode " << cookie << " cookie: " << e.what());
```

Comment:
You wrote user token into logs just now ;)

Reply:
it was masked. but when exception happens, I think that's a good idea to log whole cookie value